doc/ir-semantics.text

```diff
@@ -93,21 +98,24 @@ Update (partial) function, returns an H:
 
 ## Evaluation of simple expressions:
 
-  eval H E x = (H,E)[x]
-  eval H E lit = lit
-  eval H E primop(v, ...) = 〚primop〛(v, ...)
+  eval P H E x = (H,E)[x]
+  eval P H E lit = lit
+  eval P H E primop(v, ...) = 〚primop〛(v, ...)
+  eval P H E &&f            = &&f
```

Comment:
Should the second `&&f` be just `f`? (Maybe choosing a different letter, e.g. `g`, would be clearer.)